include/odp/api/spec/pool_types.h

```diff
@@ -456,7 +453,7 @@ typedef struct odp_pool_param_t {
 
 		/** User area size in bytes. The maximum value is defined by
 		 *  pool capability pkt.max_uarea_size. Specify as 0 if no user
-		 *  area is needed.
+		 *  area is needed. The default value is 0.
```

Comment:
Why to force default value as 0? It should be settable to any value in odp_pool_param_init().

Reply:
Maybe we could update the text a bit and say: "The minimum user area size in bytes...". So that  implementation is free to round it up to a larger value. Maybe that would simplify implementation of odp_packet_user_area_size().

Zero as default value minimizes memory usage, as applications that do not care about user area would require zero bytes automatically.

Reply:
>> "The minimum user area size in bytes..." 
Adding this line is fine.

Reply:
User area parameter spec text updated (for all four event types) in v5.